tuxemon/core/event/actions/create_npc.py

```diff
@@ -72,7 +73,7 @@ def start(self):
                 slug
             )
         else:
-            sprite = db.JSONDatabase('npc').database['npc'][slug].get('sprite_name')
+            sprite = databases.npc.database['npc'][slug].get('sprite_name')
```

Comment:
this new format is extremely obtuse.  please find a way to make it more simple:
```python
db.npc['npc'][slug].get('sprite_name')
```
would be much better.

Reply:
Latest commit should fix this. It is now just `databases.database['npc']`, a single instance of JSONDatabase() is used as was most likely intended.